autopush/websocket.py

```diff
@@ -104,7 +104,7 @@ def periodic_reporter(settings):
     settings.metrics.gauge("update.client.readers",
                            len(reactor.getReaders()))
     settings.metrics.gauge("update.client.connections",
-                           len(settings.clients))
+                        len(settings.clients))
```

Comment:
this has a few whitespace changes.. but this one looks like cruft (reduce the diff/kill please?)